examples/layout-component/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "react"]
+}
```

Comment:
I wonder why it succeeds even though `.babelrc` on the root also has these presets 🤔